src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final int CAPACITY = 100;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+
+        Task[] tasks = new Task[CAPACITY];
+        int counter = 0;
+
+        while(true) {
+            String line;
+            Scanner input = new Scanner(System.in);
+            line = input.nextLine();
+
+            if (line.equalsIgnoreCase("bye")) {
+                printByeMessage();
+                break;
+            } else if (line.equalsIgnoreCase("list")) {
+                System.out.println("____________________________________________________________");
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < counter; i++){
+                    System.out.println(i+1 + ". " + tasks[i].toString());
+                }
+                System.out.println("____________________________________________________________");
+            } else if (line.startsWith("todo")) {
+                line = line.replaceAll("todo ", "");
+                Task t = new Todo(line);
+                tasks[counter] = t;
+                printAcknowledgement(tasks[counter], counter);
+                counter++;
+            } else if (line.startsWith("deadline")) {
+                String[] words = line.split("/");
+                String description = words[0].substring(9);
+                String by = words[1].substring(3);
```

Comment:
Maybe you could replace the .substring() parameters with constants, as the 3 and 9 can be considered magic numbers.